style.css

```diff
@@ -13,66 +13,110 @@ html {
 }
 
 body {
-   margin: 4rem;
+   padding: 0 4rem;
    font-size: 2rem;
    background-color: #F5FFFA;
-   font-family: 'Loved by the King', cursive;
+   font-family: 'Open Sans', sans-serif;
 }
 
-.heading {
-   padding: 0 2rem 2rem 4rem;
- }
-
 h1 {
    font-family: 'Over the Rainbow';
+   color: #1E90FF;
    float: left;
    clear: right;
-   overflow: hidden;
-	color: #1E90FF;
+   width: 20rem;
+   position: absolute;
 }
 
-ul li {
-   font-family: 'Over the Rainbow';
+a {
+   list-style-type: none;
+   text-decoration: none;
+}
+
+/*.menu-navmain-container {
+   margin-top: 20rem;
+}*/
+
+.menu li a:hover {
+   background-color: #FFFFAA;
+   padding: 3rem;
+}
+
+.menu li a {
    color: #000;
-   float: right;
-   clear: right;
-   clear: left;
-   display: inline;
-   padding: 2rem;
-   font-size: 2rem;
 }
 
-a {
-   list-style: none;
+.menu {
+   padding-top: 4rem;
+}
+
+.menu li {
+   /*overflow: hidden;*/
+   padding: 2rem;  /* wrong way... should simply follow order */
+   list-style-type: none;
    text-decoration: none;
+   float: right;
+   /*clear: left;    clear the left of menu, can't find the class/id of main content   */
+   display: inline;
+   font-size: 1.5rem;
+   /*margin-bottom: 4rem;    space between main content   */
+   position: relative;
+   /*background-size: 4rem;*/
+   margin-bottom: 10rem; /* wrong way... should simply follow order */
```

Comment:
Yeah, try and not use `position` and use floats sparingly this will help keep elements follow order